package.json

```diff
@@ -82,13 +83,15 @@
     "jest": "^19.0.2"
   },
   "dependencies": {
+    "@paulcbetts/system-idle-time": "^1.0.4",
```

Comment:
This package is not published to the npm registry so it should include the name. Or should we include this in SFE-Packages?